app/src/main/java/com/asfoundation/wallet/my_wallets/more/MoreDialogFragment.kt

```diff
@@ -68,12 +95,80 @@ class MoreDialogFragment : BottomSheetDialogFragment(),
     }
   }
 
+  private fun buildWalletView(
+    wallet: WalletBalance,
+    isSelected: Boolean,
+    onClick: ((View) -> Unit)? = null
+  ): View = MaterialCardView(context).apply {
```

Comment:
Even tho I dont dislike the approach of configuring the view in here, I think it gets to cluttered by having to set up the color, the radius, etc.
One of the issues is that it makes it harder to be consistent, by manually adding margins and stuff like that, the other problem the the "Fragment" readability.
Maybe in the future we should create extension functions for this types of views, otherwise it gets hard to open a fragment and understand the full logic of what its happening because of all the clutter.

Reply:
Refactored